jsapp/js/components/account/dataStorage.scss

```diff
@@ -0,0 +1,55 @@
+@import "scss/_colors";
+
+.data-storage {
+  padding: 30px 40px;
+}
+
+.data-row__data {
+  display: flex;
+}
+
+.kobo-range {
+  flex: 85;
+}
```

Comment:
This is too general, as it would apply to all `kobo-range`s, similary other selectors here

Reply:
Best would be to wrap (or prefix) all these in `.data-storage`

Reply:
To prevent conflicts, I'll make these changes on the usage dashboard PR and remove these here

Reply:
tagging #3097